geode-redis/src/main/java/org/apache/geode/redis/internal/RedisCommandType.java

```diff
@@ -107,132 +107,161 @@
  */
 public enum RedisCommandType {
 
+  /***************************************
+   *** Supported Commands ***
+   ***************************************/
+
+  /*************** Keys ******************/
+
+  AUTH(new AuthExecutor(), true),
+  DEL(new DelExecutor(), true, new MinimumParameterRequirements(2)),
+  EXISTS(new ExistsExecutor(), true, new MinimumParameterRequirements(2)),
+  EXPIRE(new ExpireExecutor(), true),
+  EXPIREAT(new ExpireAtExecutor(), true),
+  FLUSHALL(new FlushAllExecutor(), true),
+  KEYS(new KeysExecutor(), true),
+  PEXPIRE(new PExpireExecutor(), true),
+  PEXPIREAT(new PExpireAtExecutor(), true),
+  RENAME(new RenameExecutor(), true),
+
+  /************* Strings *****************/
+
+  APPEND(new AppendExecutor(), true, new ExactParameterRequirements(3)),
+  GET(new GetExecutor(), true, new ExactParameterRequirements(2)),
+  SET(new SetExecutor(), true, new MinimumParameterRequirements(3)),
+
+  /************* Hashes *****************/
+
+  HGETALL(new HGetAllExecutor(), true, new ExactParameterRequirements(2)),
+  HMSET(new HMSetExecutor(), true,
+      new MinimumParameterRequirements(4).and(new EvenParameterRequirements())),
+  HSET(new HSetExecutor(), true,
+      new MinimumParameterRequirements(4).and(new EvenParameterRequirements())),
+
+  /************* Sets *****************/
+
+  SADD(new SAddExecutor(), true, new MinimumParameterRequirements(3)),
+  SMEMBERS(new SMembersExecutor(), true, new ExactParameterRequirements(2)),
+  SREM(new SRemExecutor(), true, new MinimumParameterRequirements(3)),
+
+  /********** Publish Subscribe **********/
+
+  SUBSCRIBE(new SubscribeExecutor(), true),
+  PUBLISH(new PublishExecutor(), true),
+  UNSUBSCRIBE(new UnsubscribeExecutor(), true),
+  PSUBSCRIBE(new PsubscribeExecutor(), true),
+  PUNSUBSCRIBE(new PunsubscribeExecutor(), true),
+
+  /*************** Server ****************/
+
+  PING(new PingExecutor(), true),
+  QUIT(new QuitExecutor(), true),
+  UNKNOWN(new UnknownExecutor(), true),
+
+
+  /***************************************
+   *** Unsupported Commands ***
+   ***************************************/
+
   /***************************************
    *************** Keys ******************
    ***************************************/
 
-  AUTH(new AuthExecutor()),
-  DEL(new DelExecutor(), new MinimumParameterRequirements(2)),
-  EXISTS(new ExistsExecutor(), new MinimumParameterRequirements(2)),
-  EXPIRE(new ExpireExecutor()),
-  EXPIREAT(new ExpireAtExecutor()),
-  FLUSHALL(new FlushAllExecutor()),
-  FLUSHDB(new FlushAllExecutor()),
-  KEYS(new KeysExecutor()),
-  PERSIST(new PersistExecutor()),
-  PEXPIRE(new PExpireExecutor()),
-  PEXPIREAT(new PExpireAtExecutor()),
-  PTTL(new PTTLExecutor()),
-  RENAME(new RenameExecutor()),
-  SCAN(new ScanExecutor()),
-  TTL(new TTLExecutor()),
-  TYPE(new TypeExecutor()),
+  FLUSHDB(new FlushAllExecutor(), false),
```

Comment:
Since we support FLUSHALL and FLUSHDB uses the FlushAllExecutor should we support it?

Reply:
Seems right. I've included it.